client/common/sass/components/_incident-database-card.sass

```diff
@@ -131,10 +141,11 @@
 .heading-table
 	margin: 0
 	border-bottom: 1px solid
-	border-color: $black
+	border-color: $gray-border
 	padding: 0.25rem 0 0
 	+tablet-down
-		padding: 0.5rem 0
+		padding: 0.3rem 0
+		margin-top: 1rem
```

Comment:
This adds margin top for the category details heading also. When there are multiple categories, in the database page, there is no spacing between the category details tables.

This is the design:
![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/9530293/24e6b684-bed1-44d4-8d4b-62467cd5232f)


This is how it looks after this PR:
![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/9530293/45c4bb23-d1b4-4cc1-a630-ff145e5becd4)